src/cloud/project/instance/cloud-project-instance.v6.service.js

```diff
@@ -46,7 +46,7 @@ angular.module("ovh-api-services").service("OvhApiCloudProjectInstanceV6", funct
         applicationAccess: {
             url: "/cloud/project/:serviceName/instance/:instanceId/applicationAccess",
             method: "POST",
-            isArray: true
+            isArray: false
```

Comment:
You could've also just remove the line :)

Reply:
From `src/`:
```sh
$ git grep 'isArray: false' | wc -l
186
```
LGTM ;)

Reply:
Removed ;)